client/src/pages/trainer/courses/create/index.tsx

```diff
@@ -58,15 +63,30 @@ const Create: FC = () => {
         }
         return await trigger(['name', 'category']);
       case 2:
-        /*
-            Please collect the form inputs in here, since user is at the preview section, crouseSlice should have all the necessary info we need to save in db
+        try {
+          const data = {
+            ...values,
+            category: values.category.map(({ id }) => id),
+          };
+          const res = await createCourse(data);
 
-            Please reset the courseSlice and stepperSlice to initial values once the from values are saved in db
-        */
-        return await push('/trainer/courses');
+          if ('error' in res) {
+            alertError('Failed to create course');
+          } else {
+            alertSuccess('Created courses successfully');
+            await push('/trainer/courses');
+            dispatch(reset());
+            return true;
```

Comment:
may I know why you are added `return true` in this line ?, is it expecting any boolean value to return, it was not there before